v5_MIGRATION_GUIDE.md

```diff
@@ -386,7 +386,7 @@ before the `OnResume` method of your Activity or Fragment.
 do not need to be instantiated in `OnCreate`.
 
 The PayPal integration now requires an Android App link be configured to return to your app from the 
-PayPal flow. 
+PayPal flow. See the [App Link Setup Guide](APP_LINK_SETUP.md) for more information.
```

Comment:
Instead of linking to a new markdown file do we want to link to the dev docs for app switch? I know they are currently under the v4 section which we call out, but could update to the v5 links once the docs are updated.

Reply:
It seemed a little confusing to link directly to those since they reference app switch and app links are now required on Android for any PayPal flow, not just app switch. I think for the v5 dev docs we should have a separate page for app link setup that we can refer to from any of the pages (PayPal web or app switch)

Reply:
Also like the idea of having this live in the Docs vs the repo when we start Android V5 docs!